### ISSUE TITLE
SASS iff function not working

### ISSUE THREAD

**Issue by ghost** [forumone/gesso#260]
I'm seeing the visually hidden items in Drupal after installing Gesso. I tracked it down to the _accessibility.scss file with the SASS function iff being called to see if it should be important or not. I don't see a function defining "iff" anywhere, and it seems SASS is not removing that from the compiled output so I'm seeing the following in the compiled output:

`.visually-hidden {

    clip: rect(1px, 1px, 1px, 1px) iff(true, ) !important;
    height: 1px iff(true, ) !important;
    overflow: hidden iff(true, ) !important;
    position: absolute iff(true, ) !important;
    width: 1px iff(true, ) !important;
    word-wrap: normal iff(true, ) !important;
}'

I'm still new to SASS, but, wouldn't it be @if function and not iff() function? Any help is appreciated. Thanks!

**Comment by coreylafferty** [comment]
@dutton242 You're quite right -- I recently updated the accessibility mixins in the primary branch to match those in the soon to be released 3.1 version of Gesso, but forgot to include the file with the iff function the changes depend on.  I've updated the 8.x-3.x branch.  Let me know if its working for you now, and apologies for the error.  Thank you for pointing it out!

**Comment by ghost** [comment]
That worked! Thank you!
